.gitignore

```diff
@@ -1,9 +1,9 @@
 config/*
 config2/*
-
+__pycache__
```

Comment:
Please remove any alterations to this file from this PR.

Reply:
i think i reverted this now, but not 100% proficient with git.. sorry about that